applications/apophenia.md

```diff
@@ -0,0 +1,252 @@
+
+#### Architecture
+
+The main components of the Apophenia architecture are:
+
+- **Oracle Pallet**
+- **Feed Registry Pallet**
+- **Oracle Client**
+- **Oracle Offchain Worker**
+- **TEE and Smart Contract Pools** (stretch)
+
+![Apophenia Architecture](https://github.com/subslice/oracle-parachain/blob/main/images/Paraxiom%20Pallets%20Overview.png?raw=true)
+
+### Ecosystem Fit
+
+There exists a need for a native Oracle solution within the DotSama ecosystem. Apophenia aims to fill this gap.
+
+The target audience includes parachains, smart contracts and individual accounts can register new feeds and request data from them.
+
+While there are pallets and integrations within some Parachains currently (ex: Dia on Astar or Chainlink's pallet), it remains an added complexity for the end user. Apophenia aims to provide a simple and easy to use solution for all, especially chains within the system.
+
+Apophenia also aims to provide data aggregation functionality to increase both data quality and security. This is a feature that can make Apophenia a unique solution.
+
+In regards to alternatives, we have observed that most of the [projects under Oracle are inactive](https://wiki.polkadot.network/docs/build-open-source). For those that are active, here's a quick comparison:
+
+* **interBTC**: Specialized for Bitcoin quotes, we intend to be a generalized oracle solution instead.
+* **Bridgestate**: It's focused on collateralization values of bridges.
+* **Tellor**: A full fledged solution which we are ultimately building towards, however, as it stands the consumer parachains need to be EVM compatible, have smart contracts enabled and install a separate pallet to gain oracle functionality. In our case, simply using XCM queries is sufficient to gain full functionality. We intend to use similar game-theoretic and statistical techniques to validate data across different feeds.
+* **Laminar**: Data has to be actively pushed from external sources, triggered offchain. Overall it's a pallet to be included. It has some of the same restrictions as DIA.
+* **DIA**: current solution available on Astar as ink! smart contracts. Like Laminar, data is pushed on-chain by centralized (and usually manually verified) offchain services.
+* **Acurast**: It's a multi-chain decentralized execution layer which provides a way register and run an arbitrary script (in Javascript) on the Acurast network. In that sense, it's more comparable to Phala Network and other TEE generalized solutions. While this can be used to create an Oracle prototype, it doesn't solve the problem with a data science approach.
+* **Chainlink Pallet**: It's a pallet to be included in other parachains. It's a good solution, but it's not native to the DotSama ecosystem. Another note to add is that the last commit was 2 years ago.
```

Comment:
If it's a good solution, why do we need another oracle pallet then? For example, you could simply update the current pallet instead. I'd be open to a discussion to fund something like that btw.